client/components/ItemActionConfirmation/modal.jsx

```diff
@@ -150,6 +151,10 @@ export class ItemActionConfirmationModal extends React.Component {
                 },
                 form: CancelPlanningCoveragesForm,
             },
+            [COVERAGES.ITEM_ACTIONS.CANCEL_COVERAGE.label]: {
+                title: get(COVERAGES, 'ITEM_ACTIONS.CANCEL_COVERAGE.label'),
+                form: CancelCoverageForm,
```

Comment:
This should probably have a `saveText: gettext('Cancel Coverage')`. Otherwise the button will show `SAVE`.